### ISSUE TITLE
Base.Offset is deprecated in 0.5

### ISSUE THREAD

**Issue by hayd** [BioJulia/Libz.jl#13]
```
WARNING: Base.FileOffset is deprecated, use Int64 instead.
  likely near /Users/andy/.julia/v0.5/Libz/src/zlib_h.jl:86
WARNING: Base.FileOffset is deprecated, use Int64 instead.
  likely near /Users/andy/.julia/v0.5/Libz/src/zlib_h.jl:88
WARNING: Base.FileOffset is deprecated, use Int64 instead.
  likely near /Users/andy/.julia/v0.5/Libz/src/zlib_h.jl:88
```

The suggestion being these can just be Int64. Can make PR if that seems reasonable!

**Comment by TransGirlCodes** [comment]
If this is a reasonable change, and doesn’t break tests or functionality, go for it :) !

**Comment by bicycle1885** [comment]
Solved by #14 .
